docs/general/serializers.md

```diff
@@ -369,3 +369,19 @@ class PostSerializer < ActiveModel::Serializer
   end
 end
 ```
+
+## Overriding default serializer
+
+You can use the `serializer_for` method to return a different serializer based on
```

Comment:
Can you specify that the `serializer_for` is a class method? What do you think of something like:

```diff
+If you want to define a specific serializer lookup for your associations, you can override the `ActiveModel::Serializer.serializer_for` method to return a serializer class based on the model .
```

?